src/crashstats_tools/cmd_supersearchfacet.py

```diff
@@ -289,176 +369,83 @@ def supersearchfacet(
-    # specifies a date, then we can do a single facet and print the data out
-    # and we're done
-    if period == "none" or "date" in params:
-        if "date" not in params:
-            params.update({"date": [">=%s" % start_date, "<%s" % end_date]})
-
-        if verbose:
-            console.print(f"Params: {params}")
-
-        facet_data = fetch_supersearch_facets(
-            console=console,
-            host=host,
-            params=params,
-            api_token=api_token,
-            verbose=verbose,
-        )
-
-        facet_name = params["_facets"][0]
-
-        total = facet_data["total"]
-        facets = facet_data["facets"]
-
-        if facet_name not in facets:
-            raise click.UsageError(f"{facet_name}: no data")
+    if "date" not in params:
+        params.update({"date": [">=%s" % start_date, "<%s" % end_date]})
 
-        headers = [facet_name, "count"]
-        facet_item_data = facets[facet_name]
-        records = [
-            {facet_name: item["term"], "count": item["count"]}
-            for item in sorted(facet_item_data, key=lambda x: x["count"], reverse=True)
-        ]
-
-        records.append({facet_name: "total", "count": total})
-
-        remaining = total - sum([item["count"] for item in facet_item_data])
-        if remaining:
-            records.append({facet_name: "--", "count": remaining})
-
-        if format_type == "table":
-            table = Table(show_edge=False)
-            for column in headers:
-                table.add_column(column, justify="left")
-
-            for item in records:
-                table.add_row(*[str(item[field]) for field in headers])
-            console.print(table)
-
-        elif format_type == "csv":
-            # NOTE(willkg): We need to use click.echo because console.print
-            # does rich fancy-stuff with the output
-            for line in tableize_csv(headers=headers, data=records):
-                click.echo(line)
-
-        elif format_type == "tab":
-            # NOTE(willkg): We need to use click.echo because console.print
-            # does rich fancy-stuff with the output
-            for line in tableize_tab(headers=headers, data=records):
-                click.echo(line)
-
-        elif format_type == "markdown":
-            for line in tableize_markdown(headers=headers, data=records):
-                click.echo(line)
-
-        elif format_type == "json":
-            console.print_json(json.dumps(records))
+    if verbose:
+        console.print(f"Params: {params}")
 
-        return
+    facet_data_payload = fetch_supersearch_facets(
+        console=console,
+        host=host,
+        params=params,
+        api_token=api_token,
+        verbose=verbose,
+    )
 
-    # If it is in count/period mode, then we have to do one facet for each
-    # period and compose the results.
+    if "signature" not in params["_facets"] and "signature" in facet_data_payload["facets"]:
```

Comment:
We should fix this in Crash Stats, but I'm not sure why Crash Stats does this and what might rely on it because it's been this way for *years*.